src/openapi/streaming/control-messages.ts

```diff
@@ -6,3 +6,4 @@ export const OPENAPI_CONTROL_MESSAGE_RESET_SUBSCRIPTIONS =
     '_resetsubscriptions';
 export const OPENAPI_CONTROL_MESSAGE_RECONNECT = '_reconnect';
 export const OPENAPI_CONTROL_MESSAGE_DISCONNECT = '_disconnect';
+export const OPENAPI_CONTROL_MESSAGE_PROBE = '_probe';

```

Comment:
All the other terms are pretty common/clear in streaming lingo. Could be worth a brief comment on what a probe is and what it's used for (bearing in mind this is a public repo)

Reply:
Done